python/examples/notebooks/vast.ipynb

```diff
@@ -63,6 +63,24 @@
    "execution_count": 3,
    "metadata": {},
    "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/home/andresmor/Projects/datashaper/python/examples/examples/verbs/custom_verbs.py:16: SettingWithCopyWarning: \n",
```

Comment:
Do you need to fix these paths?

Reply:
It's fine, those are warnings generated by pandas on some custom verbs